index.bs

```diff
@@ -3676,85 +3690,85 @@ the same procedure as other [=assertion signatures=] generated by the [=authenti
 
 ### Credential Backup State ### {#sctn-credential-backup}
 
-Credential backup eligibility and current backup state is conveyed by [=Authenticator data|authenticator data=] bits `3` and `4` as 
-defined in <a href="#table-authData">Table <span class="table-ref-following"/></a>.
+Credential [=backup eligibility=] and current [=backup state=] is conveyed by the `BE` and `BS` [=flags=] in the [=authenticator data=], as 
+defined in <a href="#table-authData">Table <span class="table-ref-previous"/></a>.
 
-The value of bit 3 is set during [=authenticatorMakeCredential=] operation and MUST NOT change. 
+The value of the `BE` [=flag=] is set during [=authenticatorMakeCredential=] operation and MUST NOT change.
 
-The value of bit 4 may change over time based on the current state of the [=Public Key Credential Source|credential source=]. <a href="#table-backupStates">Table <span class="table-ref-following"/></a> below defines
+The value of the `BS` [=flag=] may change over time based on the current state of the [=Public Key Credential Source|credential source=]. <a href="#table-backupStates">Table <span class="table-ref-following"/></a> below defines
 valid combinations and their meaning.
 
 <figure id="table-backupStates" class="table">
     <table class="complex data longlastcol">
             <tr>
-            <th>Bit 3 value</th>
-            <th>Bit 4 value</th>
+            <th>`BE`</th>
+            <th>`BS`</th>
             <th>Description</th>
         </tr>
         <tr>
             <td>`0`</td>
             <td>`0`</td>
             <td>
-                The credential is a single-device credential and cannot become a multi-device credential
+                The credential is a [=single-device credential=] and cannot become a [=multi-device credential=].
             </td>
         </tr>
         <tr>
             <td>`0`</td>
             <td>`1`</td>
             <td>
-                This combination is not allowed
+                This combination is not allowed.
             </td>
         </tr>
         <tr>
             <td>`1`</td>
             <td>`0`</td>
             <td>
-                The credential is currently a single-device credential but may become a multi-device credential in the future
+                The credential is currently a [=single-device credential=] but may become a [=multi-device credential=] in the future.
```

Comment:
This is still inconsistent with the Backup Eligibility definition: single- vs. multi-device-ness is defined there to be an immutable property of a credential.